flink-runtime-web/web-dashboard/src/app/interfaces/logs-bundler.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface LogsBundlerStatus {
+    status: string,
```

Comment:
I'm assuming the possible values for `status` are known, so we could be more precise than `string` here:

```
status: "PROCESSING" | "BUNDLE_READY" | "… possible other ones…";
```

With string literal types we get completion and better type-safety for where you compare the status against these values.